### PR TITLE
Destination

### PR DESCRIPTION
**Pull request by maksenius**
### Description

add destination connector

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit-connector-snowflake/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: destination/destination.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+			Default:     "",
+		},
+		config.KeyTable: {
+			Description: "A name of the table that the connector should write to.",
+			Required:    true,
+			Default:     "",
+		},
+		config.KeyPrimaryKey: {
+			Description: "A column name that used to detect if the target table" +
+				" already contains the record (destination).",
```
Comment (maha-hajja) [inline destination/destination.go:58]: maybe let the user know that it should be unique too

File: destination/destination.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+// Parameters returns a map of named sdk.Parameters that describe how to configure the Destination.
+func (d *Destination) Parameters() map[string]sdk.Parameter {
+	return map[string]sdk.Parameter{
+		config.KeyConnection: {
+			Description: "Connection string to DB2",
+			Required:    true,
+			Default:     "",
+		},
+		config.KeyTable: {
+			Description: "A name of the table that the connector should write to.",
```
Comment (maha-hajja) [inline destination/destination.go:52]:
```suggestion
			Description: "name of the table that the connector should write to.",
```

File: README.md [hunk 3]
```diff
@@ -1 +1,146 @@
...
+### Prerequisites
+
+- [Go](https://go.dev/) 1.18
+- (optional) [golangci-lint](https://github.com/golangci/golangci-lint) 1.48.0
+- (optional) [mock](https://github.com/golang/mock) 1.6.0
+
+Connector uses [go_ibm_db](https://github.com/ibmdb/go_ibm_db) library. This library required to install clidriver
+
+
+#### How to Install in Windows
```
[the remark this answers is not in the document]
Comment (maksenius) [reply README.md:17]: Yeah we have install driver anyway. go_ibm_db lib  does panic if can't find this driver on system.
License only needed for case if user wants to use specific version of db2

File: README.md [hunk 4]
```diff
@@ -1 +1,146 @@
...
+
+## Destination
+
+The DB2 Destination takes a `sdk.Record` and parses it into a valid SQL query.
+
+### Configuration Options
+
+| Name         | Description                                                                          | Required | Example                                                                 |
+|--------------|--------------------------------------------------------------------------------------|----------|-------------------------------------------------------------------------|
+| `conn`       | String line  for connection  to  DB2                                                 | **true** | HOSTNAME=localhost;DATABASE=testdb;PORT=50000;UID=DB2INST1;PWD=password |
```
[the remark this answers is not in the document]
Comment (maksenius) [reply README.md:132]: Yeah you are right, thank you. I fixed it

File: destination/destination.go (hunk 1 above)
Comment (maksenius) [reply destination/destination.go]: good idea!. I added it

**Review by maha-hajja — APPROVED** [review]
lgtm

**Pull request merged**